courses/machine_learning/deepdive2/structured/labs/1b_prepare_data_babyweight.ipynb

```diff
@@ -61,8 +61,8 @@
    "outputs": [],
```

Comment:
<p>To ensure that the bucket will default to the project if they don't replace with their bucket name, perhaps instead it can be:</p><p>os.environ["BUCKET"] = PROJECT if BUCKET == "BUCKET" else BUCKET</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>